Fix randomly breaking unit tests

### Expected behaviour
No failing of the below mentioned unit tests

### Actual behaviour
these tests are sometimes failing:
```SUMMARY:
✔ 608 tests completed
ℹ 76 tests skipped
✖ 3 tests failed
FAILED TESTS:
  Passphrase: Confirm
    ✖ should enable Next button if answer is correct
      HeadlessChrome 0.0.0 (Mac OS X 10.12.6)
    AssertionError: expected true to not equal true
        at Context.<anonymous> (webpack:///src/components/passphrase/confirm/index.test.js:85:44 <- src/tests.js:312746:56)
  Time
    ✖ shows "5 months" for the equivalent timestamp (35929631)
      HeadlessChrome 0.0.0 (Mac OS X 10.12.6)
    AssertionError: expected '48 years' to equal '5 months'
        at Context.<anonymous> (webpack:///src/components/timestamp/index.test.js:24:46 <- src/tests.js:315703:58)
  TooltipTime
    ✖ has innerHTML equal to "<span>5 months</span>" for equivalent timestamp (35929631)
      HeadlessChrome 0.0.0 (Mac OS X 10.12.6)
    AssertionError: expected '<span>48 years</span>' to equal '<span>5 months</span>'
        at Context.<anonymous> (webpack:///src/components/timestamp/index.test.js:59:44 <- src/tests.js:315734:63)
```
### Steps to reproduce
just run the tests and the second or third time you should see the failing tests

--- a/src/components/passphrase/confirm/index.test.js
+++ b/src/components/passphrase/confirm/index.test.js
@@ -101,7 +101,7 @@ describe('Passphrase: Confirm', () => {
     // for each fieldset, checks the input if its value exist in passphrase
     wrapper.find('fieldset').forEach((fieldset) => {
       fieldset.find('input').forEach((input) => {
-        if (account.passphrase.indexOf(input.props().value) > 0) {
+        if (account.passphrase.indexOf(input.props().value) > -1) {
           input.simulate('change', { target: { checked: true } });
         }
       });

